llama-index-packs/llama-index-packs-secgpt/llama_index/packs/secgpt/message.py

```diff
@@ -0,0 +1,56 @@
+"""
+In SecGPT, all messages exchanged among spokes conform to predefined formats, encapsulated within the Message class.
+"""
+import json
+
+
+class Message:
+    def function_probe_request(self, spoke_id, function):
```

Comment:
suggestion: maybe should this be a staticmethod?

Reply:
similarly for all other funcs?

Reply:
I have revised accordingly.